src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -114,7 +115,7 @@ String processClassPath(String classPath) throws IOException {
         }
 
         return new ClassPath(classPath).mapWithLibs(
-                fileDeps.getJavaFXSDKLibsPath(), "javafx-graphics", "javafx-base", "javafx-controls" );
+                fileDeps.getJavaFXSDKLibsPath(), "javafx-graphics", "javafx-base", "javafx-controls", "javafx-fxml" );
```

Comment:
Unless I'm missing something, this shouldn't be required: the change of jars is only necessary if the JavaFX module contains specific content for the target, that is not present in the Maven Central module. Is it the case?

Reply:
The javafx jars for linux don't contain all the monocle classes that are needed on Android.
It won't be needed in a next release of the SDK, as we're having both `.` and `-` separations there, but for now, this is needed.

Reply:
Sure, I only meant about adding `javafx-fxml` to this list, which shouldn't be necessary if it doesn't have any android code, which I think it is the case.